changelog.txt

```diff
@@ -11,12 +11,14 @@ next
 Added
 +++++
 
+
```

Comment:
Revert this extra newline. (Hint: use `git add -p` to add your changes one patch at a time. This helps prevent unintended changes.)
```suggestion
```